examples/wifi-echo/server/esp32/main/include/RendezvousDeviceDelegate.h

```diff
@@ -27,7 +27,7 @@ class RendezvousDeviceDelegate : public RendezvousSessionDelegate
 {
 public:
     RendezvousDeviceDelegate();
-    CHIP_ERROR Send(const char * msg);
+    CHIP_ERROR AssignedIPAddress(IPAddress & deviceAddr);
```

Comment:
The name is not clear to me as to what it does.
Could we add some description? Also wondering if this is a getter or setter. Can the reference be a const reference?